packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderItem/ConditionBuilderItem.js

```diff
@@ -117,7 +125,9 @@ export const ConditionBuilderItem = ({
       >
         <Layer>
           <h1 className={`${blockClass}__item__title`}>{title}</h1>
-          <div ref={contentRef}>{open && children}</div>
+          <div ref={contentRef} className={`${blockClass}__popover-content`}>
```

Comment:
we should wrap this div in the conditional to ensure nothing weird happens with styling since the div will still appear even with no children

Reply:
modified